test/errors_test.exs

```diff
@@ -38,7 +36,7 @@ defmodule AshGraphql.ErrorsTest do
 
     assert {:ok, result} = resp
 
-    assert %{data: nil, errors: [%{message: message}]} = result
+    assert %{data: %{"createPost" => nil}, errors: [%{message: message}]} = result
```

Comment:
Note that this change returns the behaviour to the one pre #110, so it's not breaking (there was no release between the two)